src/libraries/System.Formats.Cbor/src/PACKAGE.md

```diff
@@ -2,41 +2,121 @@
 
+Write and read primitives:
+
+```csharp
+using System.Formats.Cbor;
+
+var cborWriter = new CborWriter(CborConformanceMode.Lax);
+cborWriter.WriteTextString("Hello World");
+
+var cborReader = new CborReader(cborWriter.Encode(), CborConformanceMode.Lax);
+Console.WriteLine(cborReader.ReadTextString());
+// Hello World
+```
+
+Write and read an array:
+
+```csharp
+var cborWriter = new CborWriter(CborConformanceMode.Lax);
+cborWriter.WriteStartArray(3);
+cborWriter.WriteInt32(1);
+cborWriter.WriteInt32(2);
+cborWriter.WriteInt32(3);
+cborWriter.WriteEndArray();
+
+var cborReader = new CborReader(cborWriter.Encode(), CborConformanceMode.Lax);
+Console.WriteLine(cborReader.ReadStartArray());
+// 3
+Console.WriteLine(cborReader.ReadInt32());
+// 1
+Console.WriteLine(cborReader.ReadInt32());
+// 2
+Console.WriteLine(cborReader.ReadInt32());
+// 3
+cborReader.ReadEndArray();
+```
+
+Inspect writer and reader state:
+
+```csharp
+var cborWriter = new CborWriter(CborConformanceMode.Lax);
+cborWriter.WriteTextString("SomeArray");
+Console.WriteLine(cborWriter.BytesWritten);
+// 10
+Console.WriteLine(cborWriter.IsWriteCompleted);
+// True
+
+var cborReader = new CborReader(cborWriter.Encode(), CborConformanceMode.Lax);
+Console.WriteLine(cborReader.BytesRemaining);
+// 10
+Console.WriteLine(cborReader.ReadTextString());
+// SomeArray
+Console.WriteLine(cborReader.BytesRemaining);
+// 0
+```
+
 ## Main Types
 
 <!-- The main types provided in this library -->
 
 The main types provided by this library are:
 
-* ``
-* ``
-* ``
+* `System.Formats.Cbor.CborReader`
+* `System.Formats.Cbor.CborWriter`
+* `System.Formats.Cbor.CborReaderState`
+* `System.Formats.Cbor.CborConformanceMode`
+* `System.Formats.Cbor.CborContentException`
+* `System.Formats.Cbor.CborTag`
 
-## Addtional Documentation
+## Additional Documentation
 
 <!-- Links to further documentation. Remove conceptual documentation if not available for the library. -->
 
-* [Conceptual documentation](https://learn.microsoft.com/en-us/dotnet/standard/serialization/**LIBRARYNAME**/overview)
-* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/**LIBRARYNAME**)
+* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/system.formats.cbor)
 
 ## Related Packages
 
 <!-- The related packages associated with this package -->
 
+.NETFramework 4.6.2:
+
+* Provides HashCode types: [Microsoft.Bcl.HashCode](https://www.nuget.org/packages/Microsoft.Bcl.HashCode/)
+* Resource pooling: [System.Buffers](https://www.nuget.org/packages/System.Buffers/)
+* Efficient memory representation: [System.Memory](https://www.nuget.org/packages/System.Memory/)
+* Provides functionality over pointers: [System.Runtime.CompilerServices.Unsafe](https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/)
+* Provides underlying type for tuples: [System.ValueTuple](https://www.nuget.org/packages/System.ValueTuple/)
```

Comment:
What makes these "related"?  Based on the fact that it's using TFMs here suggests it's "dependencies", but that's already data that gets surfaced, so why would it be in the description?

I'd, personally, expect this to be more for "this is the primitive layer, but if you want a friendly, opinionated view of a thing that's easier to work with, see [something else]".  I don't think there's anything like that with CBOR, so I'd, personally, say there are no "related" packages.

Reply:
I was looking at this section more like "dependencies" packages for some reason. I am aware that user can see dependencies clicking on "Dependencies" tab.

Overall, you are right! Perhaps, I have misunderstood the purpose of this section. Are you okay if I remove the section completely? I can not list any related packages.